php/docker/templates/nginx-vhost-conf.d/75-page-cache-locations.conf

```diff
@@ -0,0 +1,53 @@
+# vim: set ft=nginx:
+
+{{- $chosenBackend := default "" .Env.STACK_PAGE_CACHE_BACKEND }}
+
+{{- if eq $chosenBackend "redis" }}
+
+location =  /.stack-cache-fetch {
+    internal;
+
+    set $redis_key $args;
+    redis_pass {{ default "localhost:6379" .Env.STACK_PAGE_CACHE_REDIS_HOST_PATH }};
+}
+
+location = /.stack-cache-store {
+    internal;
+
+    set_unescape_uri $exptime $arg_exptime;
+    set_unescape_uri $key $arg_key;
+
+    redis2_query set $key $echo_request_body;
+    redis2_query expire $key $exptime;
+    redis2_pass {{ default "localhost:6379" .Env.STACK_PAGE_CACHE_REDIS_HOST_PATH }};
```

Comment:
```suggestion
    redis2_pass {{ default "localhost" .Env.STACK_PAGE_CACHE_REDIS_HOST }}:{{ default "6379" .Env.STACK_PAGE_CACHE_REDIS_PORT }};
```

Reply:
We can work on this in a later PR, since it's not so straightforward. See:
https://github.com/openresty/srcache-nginx-module/issues/60
https://github.com/openresty/srcache-nginx-module/issues/41